salt/base/sanity.sls

```diff
@@ -5,7 +5,15 @@ niceties:
       - htop
       - traceroute
 
-time-sync:
+{% if grains["oscodename"] in ["noble"] %}
+systemd-timesyncd:
+   pkg:
+     - installed
+   service:
+     - running
```

Comment:
now that we have `grains['detect_virt']` should we guard the `running` with that to make this not error locally?

I think in order to do that you'll have to change the syntax for the docker case to be `- enabled` rather than `- enable: True` (see [salt docs](https://docs.saltproject.io/en/latest/ref/states/all/salt.states.service.html#salt.states.service.enabled)).

In total I think the result would be:

```
  service:
    {% if grains["detect_virt"] in ["docker"] %}
    - enabled
    {% else %}
    - running
    - enable: True
    {% endif %}
```

Reply:
main issue is since this service _will not_ start in docker the salt state for `running` will fail.

Reply:
doh